tdesign/src/themes/design.ts

```diff
@@ -45,11 +45,11 @@ export const theme: SplitgraphTheme = {
   },
   navbar: {
     light: "#FBFCFF",
-    dark: "#201316",
+    dark: "#1A1324",
```

Comment:
I have the impression we're not actually using this file, `design.ts` anymore. (Correct me if I'm wrong, please)
It's from when we were refactoring the old Theme UI stuff and can, I think, likely be deleted.

Reply:
I think I did a very rough search for some of the files and you might be correct. But I also didn't see any harm in changing the values, unless we want to keep them as archive?

Reply:
Sure, it's not too big of a deal. My slight preference would be to delete it, vs update it since since it's not being used in the first place these days. 
What do you think @milesrichardson?

Reply:
I think that some build step might be dependent on it existing, but it's ultimately used in pages that we're no longer deploying via this repository. So I'd suggest not deleting it, but also not worrying about updating the values in this file.